src/Sulu/Bundle/WebsiteBundle/Tests/Unit/Sulu/Bundle/WebsiteBundle/Controller/ExceptionControllerTest.php

```diff
@@ -9,11 +9,10 @@
  * with this source code in the file LICENSE.
  */
 
-namespace Sulu\Bundle\WebsiteBundle\Tests\Functional\Controller;
+namespace Sulu\Bundle\WebsiteBundle\Controller;
```

Comment:
Actually we usually set the `Tests\Functional` namespace, because the `Tests` namespace is not autoloaded in production.

Reply:
yeah the whole website bundle uses false namespace and strange nested folder. Can refractor that.

Reply:
I am not saying that you have to update everything, but here you are removing the `Tests` namespace, I think that should not be done, but kept as it was.

Reply:
@danrot oh I moved this from Functional to the Unit test namespace as its a Unit Test and not a Functional Test. And as the Unit Test Namespace for the files in there is `namespace Sulu\Bundle\WebsiteBundle\Controller;` I changed it to be consistent.

Reply:
@danrot Did refractor the website bundle test namespace.